contents/handbook/people/spending-money.md

```diff
@@ -26,14 +26,14 @@ This means expenses that are under $75 one off or under $20/month recurring that
 
 Make sure you *keep copies for all receipts*. If you expense something on a company card and cannot provide a receipt, this may be deducted from your pay.
 
-You should default to using your company card in all cases - it has no transaction fees. If you need to use your personal card in an emergency, please just let Charles know afterwards to get reimbursed manually. 
+You should default to using your company card in all cases - it has no transaction fees. If you need to use your personal card in an emergency, please post in the #ops channel in Slack afterwards and tag Charles and Grace to get reimbursed manually. 
```

Comment:
There are cases where using a personal card is required but not necessarily an emergency. For instance, if you don't live in the US you can't get a physical Brex card shipped and there's places where contactless is not available.

Reply:
Changed to 'unavoidable'